lib/buffer.js

```diff
@@ -3,12 +3,12 @@
 
 const binding = process.binding('buffer');
 const { isArrayBuffer } = process.binding('util');
+const { deprecate } = require('internal/util');
 const bindingObj = {};
 const internalUtil = require('internal/util');
 
-class FastBuffer extends Uint8Array {}
+const FastBuffer = (() => class Buffer extends Uint8Array {})();
```

Comment:
Could you remind why this change was needed? Which code breaks without it?

#11808 and #11806 are missing this, would everything be fine there?